docker-compose.override.yml

```diff
@@ -50,6 +50,9 @@ services:
     image: minio/mc:latest
     env_file:
       - .env
+    environment:
+      MINIO_ROOT_USER: ${AWS_ACCESS_KEY}
+      MINIO_ROOT_PASSWORD: ${AWS_SECRET_KEY}
```

Comment:
Why did this need to be added? We're adding the credentials as part of the init on L71 and this container doesn't exist for very long IIRC. The `s3` container should be how we access minio which _does_ need these set (and currently does have them set).

Reply:
Oops! I must have added it at some point because of some error, but it doesn't seem to be necessary now. Thanks for flagging it!